2022-04-06.VPN-Kit.md

```diff
@@ -0,0 +1,18 @@
+---
+updated: 2022-04-06
+authors:
+- name: Dave Scott
+  uri: https://github.com/djs55
+subject: 'A MirageOS VPN Kit'
+permalink: 2022-04-06.VPN-Kit
+---
+
+# A MirageOS VPN Kit
+
+[Recently, I posted](https://www.docker.com/blog/how-docker-desktop-networking-works-under-the-hood/) about how [VPN Kit](https://github.com/moby/vpnkit), built with MirageOS libraries, powers [Docker Desktop](https://www.docker.com/products/docker-desktop). Docker Desktop enables users to build and share isolated, *containerised* applications on either a Mac or Windows environment. With 15 million monthly users, it's the most popular developer tool on the planet. Hence, **MirageOS networking libraries transparently handle the traffic of millions of containers**, simplifying many developers' experience.
```

Comment:
Maybe "build and share" -> "build, share and run" (the Docker slogan)

Latest figure according to https://www.docker.com/blog/docker-raises-series-c-build-share-run/ is 16 million monthly users